CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## [Release 2.27.0](https://github.com/aws-amplify/amplify-android/releases/tag/release_v2.27.0)
+
+### Bug Fixes
+- **core:** Fix random failures in RxStorageBindingTest ([#2986](https://github.com/aws-amplify/amplify-android/issues/2986))
```

Comment:
Remove this line, it's not relevant to customers - the PR should have been marked "chore"